### ISSUE TITLE
🐛 Bug Report: WebSDK does not expose realtime types

### ISSUE THREAD

**Issue by Meldiron** [appwrite/appwrite#2417]
### 👟 Reproduction steps

 I have `appwrite.service.ts` with this function:

```
type Cursor = {
    clickPos: string;
    cursorPos: string;
    lastAction: 'none' | 'move' | 'click';
    userId: string;
} & Models.Document;

async registerListener(callback: any) {
    sdk.subscribe(`collections.${cursorCollectionId}.documents`, callback);
},
```

I do it in a separate file to keep all Appwrite logic in one place, so I don't have to initialize `new Appwrite()` each time.

Noooow... As you can see, I used the type `any` for a callback. What is the proper type? If I am not mistaken, the correct type is:

```
async registerListener(callback: (payload: RealtimeResponseEvent<Cursor>) => void) {
```

Problem is, type `RealtimeResponseEvent` is not exported from Appwrite models... Is this an incorrect solution? Do we need to export `RealtimeResponseEvent`?

![CleanShot 2021-11-14 at 14 01 22@2x](https://user-images.githubusercontent.com/19310830/141682150-e33e8d26-c144-4c52-8154-e4bc3c3f732a.png)

### 👍 Expected behavior

WebSDK does not expose real-time type, so I cannot re-use it in my code. It should, right?

### 👎 Actual Behavior

Some types are not exported. What other types are there that can cause the same problem? What was the decision behind not exporting some types?

### 🎲 Appwrite version

Version 0.11.x

### 💻 Operating system

Linux

### 🧱 Your Environment

host `demo.appwrite.io`

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by TorstenDittmann** [comment]
Let's export `RealtimeResponseEvent<T>` 👍🏻

**Comment by TorstenDittmann** [comment]
Merged => https://github.com/appwrite/sdk-generator/pull/312

**Comment by eldadfux** [comment]
Thanks for finding and listing the issue @Meldiron !

**Comment by eldadfux** [comment]
This fix is now available as part of Appwrite 0.12 which is now officially released. You can learn more at: https://dev.to/appwrite/its-here-announcing-the-release-of-appwrite-012-5c8b
